roles/teuthology/vars/zypper_opensuse_leap_15.3.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Why is this file in this PR?

Reply:
I was testing it on the latest distro, so this is a complement.

Reply:
Ah, right, you do a full pulpito/paddles/teuthology dev setup to test, right?

Reply:
@djgalloway exactly